test/io/ferguson/recordeof.good

```diff
@@ -0,0 +1 @@
+Done
```

Comment:
This test would be more convincing if it actually wrote and read at least one record....